### PR TITLE
link to list of supported buildpacks, rather than maintaining our own

### PR DESCRIPTION
**Pull request by afeld**
/cc @jameshupp @mogul

### CODE REVIEW

**Review by jameshupp — APPROVED** [review]

**Comment by brittag** [comment]
Just a thought: it might be odd to ask readers to run `cf buildpacks` here, since most of the people reading this probably don't have access to cloud.gov yet. We may want to just recommend looking at that CF docs page for the up-to-date list.

**Comment by afeld** [comment]
Ok, went a bit further with this — see the commit messages.

File: content/overview/pricing/whats-included.md [hunk 1]
```diff
@@ -2,11 +2,14 @@
 menu:
   overview:
     parent: pricing
-title: What’s included
+title: What’s included?
 weight: 10
 aliases:
   - /docs/intro/pricing/whats-included
   - /intro/pricing/whats-included
+  - /docs/intro/pricing/paying-for-otherservices
```
Comment (jcscottiii) [inline content/overview/pricing/whats-included.md:10]: a couple of things:
there's three new `paying-for-otherservices` urls. do we want to add a third `whats-included` (`/overview/pricing/paying-for-other-services`)
also two of these have no hyphen between `otherservices` and there's `other-services`. i think having the hyphen between each word helps to be consistent

Comment (afeld) [reply content/overview/pricing/whats-included.md:10]: The aliases are only there to redirect from the old URLs for the deleted page, none of which had a hyphen.

File: content/overview/technology/responsibilities.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Here's a chart to illustrate this in three example use cases:
 
 App #1 uses a [Cloud Foundry standard buildpack]({{< relref "docs/getting-started/concepts.md#buildpacks" >}}). (A buildpack provides support for a programming language.) The customer is only responsible for the app code and its dependencies.
 
-App #2 uses an unsupported/custom buildpack, so the customer's responsibility expands from the app code to managing the unsupported buildpack and its dependencies. If you choose to use a custom buildpack, you are responsible for:
+App #2 uses an unsupported/custom buildpack, so the customer's responsibility expands from the app code to managing the custom buildpack and its dependencies. If you choose to use a custom buildpack, you are responsible for:
```
Comment (jcscottiii) [inline content/overview/technology/responsibilities.md]: `managing the custom buildpack` -> `managing the unsupported/custom buildpack` to be consistent. thoughts?

File: content/overview/pricing/whats-included.md (hunk 1 above)
Comment (jcscottiii) [reply content/overview/pricing/whats-included.md:10]: > The aliases are only there to redirect from the old URLs for the deleted page, none of which had a hyphen.

ohhh!! gotcha

File: content/overview/technology/responsibilities.md (hunk 2 above)
Comment (afeld) [reply content/overview/technology/responsibilities.md]: Done!

**Review by jcscottiii — APPROVED** [review]

**Pull request merged**